src/OBJFile.js

```diff
@@ -138,13 +138,13 @@ class OBJFile {
       const vertexString = lineItems[i + 1];
       const vertexValues = vertexString.split('/');
 
-      if (vertexValues.length < 1 || vertexValues.length > 3) { throw (`Two many values (separated by /) for a single vertex${this.filePath}${this.lineNumber}`); }
+      if (vertexValues.length < 1 || vertexValues.length > 3) { throw (`Too many values (separated by /) for a single vertex${this.filePath}${this.lineNumber}`); }
 
       let vertexIndex = 0;
       let textureCoordsIndex = 0;
       let vertexNormalIndex = 0;
       vertexIndex = parseInt(vertexValues[0]);
-      if (vertexValues.length > 1 && (!vertexValues[1] == '')) { textureCoordsIndex = parseInt(vertexValues[1]); }
+      if (vertexValues.length > 1 && (vertexValues[1] != '')) { textureCoordsIndex = parseInt(vertexValues[1]); }
```

Comment:
Note that coincidentally this works in both cases:

`
!"5" == ''
returns true
!"" == ''
and returns false as needed.`

So there's no actual reproducible bug, (test cases confirm that it works with texture coords present and not present).
But I agree this is a nice pair of cleanups.

Thanks for submitting the PR! I will post another comment once the fix has been published to npm.

Reply:
Just published to npm (code changes are included as of version: 0.5.2)

Reply:
Correction:  published to npm, but included as of version 0.5.3